Reminder-App/Reminder-App/Reminder.swift

```diff
@@ -8,13 +8,17 @@
 import Foundation
 
 struct Reminder {
+    
+    // MARK:- Instance Properties
```

Comment:
if you want to use - after semicolon add a space before -
this is a suggesstion if you don't want you can skip this comment.